File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_Three/Java/prog5.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		System.out.println("Enter num : ");
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		for(int i = 2; i <= num / 2; i++) {
			System.out.print(num % i == 0 ? i + " " : "");
		}
	}
}

File: Python Code/Daily_Flash_Code_0/Week 2/24Jan/10-DailyFlash_Solutions/10-DailyFlash_Solutions/24_Jan_Solutions_Three/C/prog5.c
#include<stdio.h>

int main(void) {
	int num;
	printf("Enter num : ");
	scanf("%d", &num);

	for(int i = 2; i <= num / 2; i++) {
		if(num % i == 0)
			printf("%d ", i);
	}

	return 0;
}
